web/js/codeworld.js

```diff
@@ -158,6 +158,63 @@ function initCodeworld() {
     }
 }
 
+var cStream;
+var recorder;
+var chunks = [];
```

Comment:
Doesn't this need to be cleared every time you start a recording?  It looks like the way this is now, you'll keep building more onto the data stream, but re-save the previous recorded content.